test/integration/framework/etcd.go

```diff
@@ -84,8 +82,7 @@ func startEtcd() (func(), error) {
 		return nil, err
 	}
 
-	etcdURL = currentURL
-	os.Setenv("KUBE_INTEGRATION_ETCD_URL", etcdURL)
+	os.Setenv("KUBE_INTEGRATION_ETCD_URL", currentURL)
```

Comment:
we set the var here, so your plan is to do this in your code, right?

```
currentURL, stop, err := RunCustomEtcd("integration_test_etcd_data", nil)
	if err != nil {
		return nil, err
	}

	os.Setenv("KUBE_INTEGRATION_ETCD_URL", currentURL)
```

Reply:
@aojea yep!